brooklyn-tosca-transformer/src/main/java/io/cloudsoft/tosca/a4c/brooklyn/ToscaPolicyDecorator.java

```diff
@@ -0,0 +1,14 @@
+package io.cloudsoft.tosca.a4c.brooklyn;
+
+import java.util.Map;
+import java.util.Set;
+
+import com.google.common.base.Optional;
+
+public interface ToscaPolicyDecorator {
+
+    public static final String POLICY_FLAG_TYPE = "type";
+    public static final String POLICY_FLAG_NAME = "name";
+
+    void decorate(Map<String, ?> policyData, String policyName, Optional<String> type, Optional<Set<String>> groupMembers);
```

Comment:
As per one of the previous comments I think it would be better to drop the `Optional` and just test whether the set is empty or not. Don't want it to hold up this pull request though.